src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private List<Bulldozer> bulldozerList = new ArrayList<>();
+
+    @Override
+    public List<Bulldozer> get() {
```

Comment:
You should return new List of buldozers every time this method is invoked. Don't use "bulldozerList" as class field.
Also fix it in other Producers